src/Surfnet/ServiceProviderDashboard/Application/Command/Entity/UpdateEntityAclCommand.php

```diff
@@ -34,7 +34,7 @@ class UpdateEntityAclCommand implements Command
      * @var IdentityProvider[]
      * @Assert\All({
      *     @Assert\NotBlank(),
-     *     @Assert\Type(type="Surfnet\ServiceProviderDashboard\Domain\ValueObject\IdentityProvider")
+     *     @Assert\Type(type="Surfnet\ServiceProviderDashboard\Domain\Entity\IdentityProvider")
```

Comment:
Good catch, strange this did not result in issues during my test session.

Reply:
Probably because of the initially missing `isValid` check.